### PR TITLE
[zh] translate kubeadm commands which are missing in zh version

### PR DESCRIPTION
**Pull request by howieyuen**
ref: https://github.com/kubernetes/website/issues/24486

this PR contains all files in docs/reference/setup-tools/kubeadm/generated
implementation-details.md will committed in other PR.

### CODE REVIEW

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha_certs.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+<col span="1" style="width: 10px;" />
+<col span="1" />
+</colgroup>
+<tbody>
+
+<tr>
+<td colspan="2">-h, --help</td>
+</tr>
+<tr>
+<td></td><td style="line-height: 130%; word-wrap: break-word;">help for certs</td>
```
Comment (tengqm) [inline content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha_certs.md:24]:
```suggestion
<td></td><td style="line-height: 130%; word-wrap: break-word;"><!-- help for certs-->certs 命令的帮助</td>

```

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha_certs_generate-csr.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
+
+<!-- ### Synopsis -->
+### 概要
+
+<!-- 
+Generates keys and certificate signing requests (CSRs) for all the certificates required to run the control plane. This command also generates partial kubeconfig files with private key data in the  "users &gt; user &gt; client-key-data" field, and for each kubeconfig file an accompanying ".csr" file is created.
+-->
+为运行控制平面的所有证书生成密钥和证书签名请求（CSR）。该命令会生成部分 kubeconfig 文件，
```
Comment (tengqm) [inline content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha_certs_generate-csr.md:8]:
```suggestion
为运行控制平面所需的所有证书生成密钥和证书签名请求（CSR）。该命令会生成部分 kubeconfig 文件，

```

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha_certs_generate-csr.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+<!-- 
+Generates keys and certificate signing requests (CSRs) for all the certificates required to run the control plane. This command also generates partial kubeconfig files with private key data in the  "users &gt; user &gt; client-key-data" field, and for each kubeconfig file an accompanying ".csr" file is created.
+-->
+为运行控制平面的所有证书生成密钥和证书签名请求（CSR）。该命令会生成部分 kubeconfig 文件，
+其中 "users &gt; user &gt; client-key-data" 字段包含私钥数据，并为每个 kubeconfig 文件创建一个随附的".csr"文件。
+
+<!--  
+This command is designed for use in [Kubeadm External CA Mode](https://kubernetes.io/docs/tasks/administer-cluster/kubeadm/kubeadm-certs/#external-ca-mode). It generates CSRs which you can then submit to your external certificate authority for signing.
+-->
+该命令设计用于[Kubeadm 外部 CA 模式](https://kubernetes.io/docs/tasks/administer-cluster/kubeadm/kubeadm-certs/#external-ca-mode)。
```
Comment (tengqm) [inline content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha_certs_generate-csr.md:14]:
```suggestion
该命令设计用于 [Kubeadm 外部 CA 模式](/zh/docs/tasks/administer-cluster/kubeadm/kubeadm-certs/#external-ca-mode)。 

```

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha_certs_generate-csr.md [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+<!-- <td></td><td style="line-height: 130%; word-wrap: break-word;">Path to a kubeadm configuration file.</td> -->
+<td></td><td style="line-height: 130%; word-wrap: break-word;">kubeadm 配置文件的路径。</td>
+</tr>
+
+<tr>
+<td colspan="2">-h, --help</td>
+</tr>
+<tr>
+<!-- <td></td><td style="line-height: 130%; word-wrap: break-word;">help for generate-csr</td> -->
+<td></td><td style="line-height: 130%; word-wrap: break-word;">生成 csr 的帮助命令</td>
```
Comment (tengqm) [inline content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha_certs_generate-csr.md:72]:
```suggestion
<td></td><td style="line-height: 130%; word-wrap: break-word;">generate-csr 命令的帮助</td>

```

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha_certs_generate-csr.md [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+</tr>
+
+<tr>
+<td colspan="2">--kubeconfig-dir string&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Default: "/etc/kubernetes"</td>
```
Comment (tengqm) [inline content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha_certs_generate-csr.md:76]: Default 未译

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_init_phase_kubelet-finalize.md [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+
+```
+kubeadm init phase kubelet-finalize [flags]
+```
+
+<!-- ### Examples -->
+### 示例
+
+```
+  # Updates settings relevant to the kubelet after TLS bootstrap"
```
Comment (tengqm) [inline content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_init_phase_kubelet-finalize.md:16]: 注释未译

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_init_phase_kubelet-finalize_all.md [hunk 7]
```diff
@@ -0,0 +1,88 @@
...
+### 示例
+
+<!--  
+```
+  # Updates settings relevant to the kubelet after TLS bootstrap
+  kubeadm init phase kubelet-finalize all --config
+```
+-->
+```
+  ＃在 TLS 引导后更新与 kubelet 相关的设置
```
Comment (tengqm) [inline content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_init_phase_kubelet-finalize_all.md:23]: 这里的井号是不是全角？

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_init_phase_kubelet-finalize_all.md [hunk 8]
```diff
@@ -0,0 +1,88 @@
...
+
+   <table style="width: 100%; table-layout: fixed;">
+<colgroup>
+<col span="1" style="width: 10px;" />
+<col span="1" />
+</colgroup>
+<tbody>
+
+<tr>
+<td colspan="2">--cert-dir string&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Default: "/etc/kubernetes/pki"</td>
```
Comment (tengqm) [inline content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_init_phase_kubelet-finalize_all.md:38]: Default

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_init_phase_kubelet-finalize_experimental-cert-rotation.md [hunk 9]
```diff
@@ -0,0 +1,75 @@
...
+
+   <table style="width: 100%; table-layout: fixed;">
+<colgroup>
+<col span="1" style="width: 10px;" />
+<col span="1" />
+</colgroup>
+<tbody>
+
+<tr>
+<td colspan="2">--cert-dir string&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Default: "/etc/kubernetes/pki"</td>
```
Comment (tengqm) [inline content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_init_phase_kubelet-finalize_experimental-cert-rotation.md:25]: Default

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_upgrade_node_phase_preflight.md [hunk 10]
```diff
@@ -0,0 +1,69 @@
+
+<!-- ### Synopsis -->
+### 概要
+
+
+<!-- Run pre-flight checks for kubeadm upgrade node. -->
+
+运行 kubeadm 升级节点的飞行前检查。
```
Comment (tengqm) [inline content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_upgrade_node_phase_preflight.md:8]:
```suggestion
运行 kubeadm 升级节点的预检。
```

preflight check 一直没有一种好的译法。直译成’飞行前‘似乎不妥。

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha_certs.md (hunk 1 above)
Comment (howieyuen) [reply content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha_certs.md]: oops, missed one

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha_certs_generate-csr.md (hunk 3 above)
Comment (howieyuen) [reply content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha_certs_generate-csr.md]: omg...I must keep a sharp mind...

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha_certs_generate-csr.md (hunk 4 above)
Comment (howieyuen) [reply content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha_certs_generate-csr.md]: it's a proper noun, missed two

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_init_phase_kubelet-finalize.md (hunk 6 above)
Comment (howieyuen) [reply content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_init_phase_kubelet-finalize.md]: missed three...

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_init_phase_kubelet-finalize_all.md (hunk 7 above)
Comment (howieyuen) [reply content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_init_phase_kubelet-finalize_all.md]: yes, stupid mistake +1 again

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha_certs_generate-csr.md (hunk 2 above)
Comment (howieyuen) [reply content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha_certs_generate-csr.md]: requested

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_upgrade_node_phase_preflight.md (hunk 10 above)
Comment (howieyuen) [reply content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_upgrade_node_phase_preflight.md]: Yes, I look up some materials and translation website, it has no proper translation indeed.

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha_certs_generate-csr.md (hunk 5 above)
Comment (howieyuen) [reply content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha_certs_generate-csr.md]: missed four

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_init_phase_kubelet-finalize_all.md (hunk 8 above)
Comment (howieyuen) [reply content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_init_phase_kubelet-finalize_all.md]: no.5

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_init_phase_kubelet-finalize_experimental-cert-rotation.md (hunk 9 above)
Comment (howieyuen) [reply content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_init_phase_kubelet-finalize_experimental-cert-rotation.md:25]: No.6 ignore

**Pull request merged**